cmd/main.go

```diff
@@ -0,0 +1,162 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"os/user"
+	"path"
+	"runtime"
+	"strings"
+
+	"github.com/google/go-github/github"
+)
+
+func IsInstalled() bool {
```

Comment:
This should return bool and error, because you might have an error but it can be something else than "not exists" and in that case you ignore the error. Also it should take in the location as input arg which can be empty in that case you check default.